dev/ci/README.md

```diff
@@ -178,6 +178,11 @@ versions of Coq (32bit and 64bit).
 The Windows jobs are enabled on Coq's repository, where pipelines for
 pull requests run.
 
+If the additional secret variable `WINDOWS_ALL_ADDONS` is set to `enabled`,
+an extended set of addons will be added to the Windows installer.
+This leads to a considerable runtime in CI and is disabled by default for
+pipelines for pull requests.
+
```

Comment:
Two tags for a ternary switch may become problematic, I'd suggest you reuse the existing `WINDOWS` variable here.

Reply:
@ejgallego : I am not sure if this would get easier, because both switches are possibly controlled in different places and it might get tricky to not overwrite one setting with another. But I don't know how the WINDOWS switch is really used, so I can't really tell if this would be a problem or not. If the point of control would be the same, for sure I can change this.

Reply:
I mean where and how are these variables set? The default value in the gitlab settings might depend on environemnts and also on specific branch properties (protected branches ...). As far as I can tell if something is a signed release build or not is controlled here. But I guess most settings are controlled in coqbot. Not sure if there are other entities which control these variables.

One might want to control WINDOWS_ALL_ADDONS at least partly in a similar way as the release signign key variable. But I guess WINDOWS is always controlled by coqbot. You see my point?

Reply:
I am not sure I follow, `WINDOWS` is set as a regular variable, AFAICT `coqbot` has nothing to do with it. It makes sense to me that the `WINDOWS` variable becomes `::= enabled | enabled_with_addons` etc...

Reply:
I guess my concerns are a result of a lack of understanding of details on how these variables are set.  I will change it then.

Reply:
TTOMK the variables are set from the GitLab CI panel, but ccing @Zimmi48 and @SkySkimmer to be more sure about that.

Reply:
Anyways my comment is just a suggestion, we can use two variables if we think it is better [but I tend to dislike this 3 settings 2 binary variables because you find situations such as `WINDOWS=disabled` but `WINDOWS_ALL_ADDONS=enabled`.

In fact, maybe the variable should just be `WINDOWS_ADDONS=list_of_addons` so people could customize the build easily.

Reply:
>TTOMK

What?

>the variables are set from the GitLab CI panel

yes

Reply:
> TTOMK

To The Best of My Knowledge I meant ...

Reply:
@ejgallego : I want to have the small and large set defined somewhere in GIT, because changes to this are not trivial. Usually changes to the set go together with other changes in GIT. Decoupling this would not be good.

It is by the way easy to do a local build with a custom set of addons (e.g. for teachers).

Reply:
@ejgallego : I just changed it. Unfortunately gitlab doesn't allow boolean expressions in the .yml file, so I had to use regexp for the enable test. Please review if you really like this better.